project/Dependencies.scala

```diff
@@ -85,13 +85,16 @@ object Dependencies {
     "io.circe" %% "circe-parser"
   ).map(_ % circeV)
 
+  val json4sNative = "org.json4s" %% "json4s-native" % "3.6.1" % "test"
```

Comment:
This gives us a nice syntax for building JSON data to use as test expectations. See `MockSam`.